src/middleware/auth.js

```diff
@@ -164,6 +164,7 @@ const remoteAuthentication = ({signatureValidator, entry}) => {
       host:             host.name,
       port:             parseInt(port, 10),
       authorization:    req.headers.authorization,
+      sourceIp:         req.ip,
```

Comment:
we have 'trust proxy' enabled in Express, so this should give us the right data.